conf/messages

```diff
@@ -686,8 +686,6 @@ alertBannerShuttering.repayments = Repayments
 alertBannerShuttering.taxcalc = Tax Calculation
 alertBannerShuttering.fandf = Trusted Helpers
 alertBannerShuttering.childBenefit = Child Benefit
```

Comment:
Do we need child benefit property, as you removed that as well from shuttering view

Reply:
Child benefit is still there, it just moved higher up so its in alphabetical order